examples/StandardFirmataWiFi/wifiConfig.h

```diff
@@ -30,16 +30,16 @@
  */
 //#define WIFI_101
 
-//do not modify the following 10 lines
+//do not modify the following 11 lines
 #if defined(ARDUINO_SAMD_MKR1000) && !defined(WIFI_101)
 // automatically include if compiling for MRK1000
 #define WIFI_101
 #endif
 #ifdef WIFI_101
 #include <WiFi101.h>
-#include "utility/WiFiStream.h"
-WiFiStream stream;
-#define WIFI_LIB_INCLUDED
+#include "utility/WiFiClientStream.h"
```

Comment:
What is the memory overhead of including both client and server streams simultaneously?